### ISSUE TITLE
Research pyt

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-django-template#325]
I have found an interesting project: https://github.com/python-security/pyt
It is focused on security.

Maybe it will be worth adding?

**Comment by DmytroLitvinov** [comment]
Hm.. Looks very attractive.
What do you think of adding some tools for formatting like https://github.com/ambv/black?

**Comment by sobolevn** [comment]
We should revisit this in several month. Currently there are several drawbacks:
1. no docs
2. https://github.com/python-security/pyt/issues/127

**Comment by KevinHock** [comment]
Woah woah woah 😁 You can say all the bad things you want about lack of -r option, and several other things, but docs, I got docs, see README in https://github.com/python-security/pyt/tree/re_organize_code/pyt and http://pyt.readthedocs.io/en/latest/?badge=latest plus the many page thesis. 

But yeah, I'll ping you folks once -r is done ✅

**Comment by KevinHock** [comment]
(That README links to other READMEs namely https://github.com/python-security/pyt/blob/re_organize_code/pyt/cfg/README.rst and https://github.com/python-security/pyt/blob/re_organize_code/pyt/analysis/README.rst at the moment.)

**Comment by DmytroLitvinov** [comment]
Just leave it here:
https://github.com/PyCQA/bandit

**Comment by sobolevn** [comment]
@DmytroLitvinov have you tried it? Can you please open a new issue?

**Comment by DmytroLitvinov** [comment]
@sobolevn Not yet.
